tests/migration/recovery.go

```diff
@@ -0,0 +1,239 @@
+	fakeMigrationSuccessInPVC := func(virtClient kubecli.KubevirtClient, pvcName, namespace string) {
+		var err error
+
+		By("Creating a job")
+		fakeSuccessJob := &batchv1.Job{
+			ObjectMeta: k8smeta.ObjectMeta{
+				GenerateName: "migration-success-faker-",
+			},
+			Spec: batchv1.JobSpec{
+				ActiveDeadlineSeconds:   pointer.P(int64(90)),
+				BackoffLimit:            pointer.P(int32(1)),
+				TTLSecondsAfterFinished: pointer.P(int32(90)),
+				Template: k8score.PodTemplateSpec{
+					ObjectMeta: k8smeta.ObjectMeta{
+						GenerateName: "backend-storage-recover-",
+					},
+					Spec: k8score.PodSpec{
+						RestartPolicy: k8score.RestartPolicyNever,
+						SecurityContext: &k8score.PodSecurityContext{
+							RunAsNonRoot: pointer.P(true),
+							RunAsUser:    pointer.P(int64(util.NonRootUID)),
+							RunAsGroup:   pointer.P(int64(util.NonRootUID)),
+							FSGroup:      pointer.P(int64(util.NonRootUID)),
+							SeccompProfile: &k8score.SeccompProfile{
+								Type: k8score.SeccompProfileTypeRuntimeDefault,
+							},
+						},
+						Containers: []k8score.Container{{
+							Name: "container",
+							SecurityContext: &k8score.SecurityContext{
+								AllowPrivilegeEscalation: pointer.P(false),
+								Capabilities:             &k8score.Capabilities{Drop: []k8score.Capability{"ALL"}},
+							},
+							Image:   libregistry.GetUtilityImageFromRegistry("vm-killer"), // Any image will do, we just need `touch`
+							Command: []string{"touch"},
+							Args:    []string{"/meta/migrated"},
+							VolumeMounts: []k8score.VolumeMount{{
+								Name:      "backend-storage",
+								MountPath: "/meta",
+								SubPath:   "meta",
+							}},
+						}},
+						Volumes: []k8score.Volume{{
+							Name: "backend-storage",
+							VolumeSource: k8score.VolumeSource{
+								PersistentVolumeClaim: &k8score.PersistentVolumeClaimVolumeSource{
+									ClaimName: pvcName,
+								},
+							},
+						}},
+					},
+				},
+			},
+		}
+
+		fakeSuccessJob, err = virtClient.BatchV1().Jobs(namespace).Create(context.Background(), fakeSuccessJob, k8smeta.CreateOptions{})
+		Expect(err).NotTo(HaveOccurred())
+
+		By("Waiting for the job to succeed")
+		err = job.WaitForJobToSucceed(fakeSuccessJob, time.Minute)
+		Expect(err).NotTo(HaveOccurred())
+
+		By("Removing the job")
+		// Job is auto-removed after 90 seconds, might already be gone, deleting anyway to free PVC
+		_ = virtClient.BatchV1().Jobs(namespace).Delete(context.Background(), fakeSuccessJob.Name, k8smeta.DeleteOptions{
+			PropagationPolicy: pointer.P(k8smeta.DeletePropagationBackground),
+		})
+	}
+
+	DescribeTable("should successfully differ a migration", func(fakeSuccess bool, flakeCheck bool) {
+		virtClient, err := kubecli.GetKubevirtClient()
+		Expect(err).NotTo(HaveOccurred())
+
+		if flakeCheck {
+			// Skip the test meant for the flake-checker if we're not currently flake-checking
+			if os.Getenv("KUBEVIRT_FLAKE_CHECK") == "" {
```

Comment:
Pardon my ignorance, could we filter on labels instead?

Reply:
I'm not sure what you mean... But please elaborate, my solution doesn't appear to work properly anyway!
What I need is to run my tests as parallel ones in sig-compute, and serial + CR mod in flake-checker.
That env var doesn't seem to make it to the test, but adding a -opt to the main test binary should do the trick.

Reply:
I was thinking of filtering the tests by focussing on a specific label instead of using `Skip` in the tests. (e.g. something like `export KUBEVIRT_LABEL_FILTER='(flakecheck)'`)

Reply:
Good point, that should work... It does mean adding `!flakecheck` to all other lanes and writing a more complex filter for the flake-check lane, but it might still be better than my whole scheme, especially with the Skip there... I'll give it a shot, thanks!